.github/workflows/nightly-android-testbuild.yml

```diff
@@ -1,78 +1,81 @@
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
+  # check_date:
+  #   runs-on: ubuntu-latest
+  #   name: Check latest commit
+  #   outputs:
+  #     should_run: ${{ steps.commitswithintime.outputs.has-new-commits-within-time }}
+  #   steps:
+  #     - uses: actions/checkout@v4
+  #       with:
+  #         fetch-depth: 0
 
-      - name: Run CommitsWithinTime script
-        uses: AlexHolderDeveloper/CommitsWithinTime@v1.1.11
-        id: commitswithintime
-        with:
-          days: 1
-      - name: Get the output from CommitsWithinTime
-        run: |
-          echo "The 'has-new-commits-within-time' value is ${{ steps.commitswithintime.outputs.has-new-commits-within-time }}"
-          echo "The 'number-of-commits-within-time' value is ${{ steps.commitswithintime.outputs.number-of-commits-within-time }}"
-          echo "The 'total-commits' value is ${{ steps.commitswithintime.outputs.total-commits }}"
+      # - name: Run CommitsWithinTime script
+      #   uses: AlexStormwood/CommitsWithinTime@v1.1.12
+      #   id: commitswithintime
+      #   with:
+      #     days: 1
+      # - name: Get the output from CommitsWithinTime
+      #   run: |
+      #     echo "The 'has-new-commits-within-time' value is ${{ steps.commitswithintime.outputs.has-new-commits-within-time }}"
+      #     echo "The 'number-of-commits-within-time' value is ${{ steps.commitswithintime.outputs.number-of-commits-within-time }}"
+      #     echo "The 'total-commits' value is ${{ steps.commitswithintime.outputs.total-commits }}"
   node-deb-gh-build-play:
-    # The type of runner that the job will run on
-    # https://github.com/Apple-Actions/Example-iOS/blob/master/.github/workflows/test.yml
     runs-on: ubuntu-latest
-    needs: check_date
-    if: ${{ needs.check_date.outputs.should_run == 'true' }}
+    # needs: check_date
+    # if: ${{ needs.check_date.outputs.should_run == 'true' }}
     continue-on-error: false
     strategy:
       matrix:
         node-version: ['20']
-    outputs:
-      prnum: ${{ steps.prnum.outputs.value }}
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Configure Turborepo Remote Cache
+        run: |
+          mkdir -p .turbo
+          if [ -n "${{ vars.TURBO_API_URL }}" ]; then
+            echo '{"apiurl": "${{ vars.TURBO_API_URL }}"}' > .turbo/config.json
+            echo "Turborepo config file created at .turbo/config.json"
+          else
+            echo "TURBO_API_URL not set. Skipping Turborepo cache configuration."
+          fi
+          if [ -n "$TURBO_TEAM" ] && [ -n "$TURBO_TOKEN" ]; then
+            echo "TURBO_TEAM and TURBO_TOKEN environment variables set"
+          else
+            echo "TURBO_TEAM and/or TURBO_TOKEN not set. Remote caching may not be available."
+          fi
```

Comment:
You'll need to add TURBO_TOKEN and TURBO_TEAM to env variables exposed in the workflow at top level. See the lint script for example.

Reply:
Thanks